doc/UserGuide/tables/UPP_GRIB2_Table_byID.csv

```diff
@@ -241,6 +241,7 @@ Canopy conductance - soil component,RCSOL_ON_SURFACE,RCSOL,241
 Potential evaporation,PEVPR_ON_SURFACE,PEVPR,242
 Heat diffusivity on sigma surface,VEDH_ON_SIGMA_LVLS,VEDH,243
 Accumulated large scale snow,Not currently available for grib2,NA,244
+Time-maximum composite reflectivity,MAX_REFC_ON_ENTIRE_ATMOS,MAXREFC,244
```

Comment:
@selbigmtnwx23 The addition looks good, but you would need to remove line 243 so that there isn't duplicate info for ID# 244. (And then also make the addition Ben mentioned for ID 248)